ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/worldview/BonsaiWorldStateUpdateAccumulator.java

```diff
@@ -290,6 +299,19 @@ public void commit() {
               final BonsaiAccount updatedAccount;
               final BonsaiValue<BonsaiAccount> updatedAccountValue =
                   accountsToUpdate.get(updatedAddress);
+
+              final Map<StorageSlotKey, BonsaiValue<UInt256>> pendingStorageUpdates =
```

Comment:
we need to fullsync some nodes to be sure we are not adding a consensus issue 
(goerli, sepolia, and maybe etc at the minimum)

Reply:
I snap synced a goerli node a few days ago and it has been going well.  I will start a full sync on goerli and another on sepolia and let them run over the weekend

Reply:
could you try to run a mainnet snapsync with the heal flat db flag ? I see that this PR is modifying a lot the stream of the flat db datas and it will be nice if can be sure that it's ok